src/patternfly/components/Chips/chip.scss

```diff
@@ -0,0 +1,131 @@
+@import "../../patternfly-utilities";
+
+.pf-c-chip {
+  --pf-c-chip--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip--MarginTop: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-chip--MarginBottom: calc(#{var(--pf-global--spacer--xs)} / 2);
```

Comment:
Components should not margin themselves because by definition, they are supposed to take all available space. In this case, the chip can be spaced out by the parent chip group.